HC-941_Can I track my order? (#26)
arduino/help-center-content: content/Store Support/General Store/Can-I-track-my-order.md

Please note that orders shipped to **NON-US** or **NON-EU** countries may be subject to local import taxes (VAT and Duties), and that local courier companies may charge a service fee in addition to the original shipping costs.

> Please, notice that NON-US orders shipped via DHL Parcel International Standard or Asendia are going to be delivered by the local postal service. In this case, many countries create a **local tracking number** for these parcels. It can be found in the **right corner**  at the page of your international tracking number.

Reviewer: ```suggestion
Please note that non-US orders shipped via DHL Parcel International Standard or Asendia will be delivered by the local postal service. In this case, many countries create a **local tracking number** for these parcels. It can be found in the **right corner**  at the page of your international tracking number.
```

> ![Ascendia order receipt with local tracking number highlighted](img/AscendiaLocalTrackingNumber.png)

Reviewer: ```suggestion
![Asendia order receipt with local tracking number highlighted](img/AscendiaLocalTrackingNumber.png)
```
